### PR TITLE
feat(index): add theme section

### PR DESCRIPTION
**Pull request by SukkaW**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

Add theme section to index.

image

### CODE REVIEW

**Comment by curbengh** [comment]
I can see where https://github.com/hexojs/site/pull/1141 and this PR are inspired from. The description (especially the keywords) definitely needs to be updated.

**Comment by SukkaW** [comment]
@curbengh I am going to add another section about API in a separate PR. But I am not sure what background color should I use in that sections.

File: themes/navy/languages/en.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ menu:
 
 index:
   get_started: Get started
+  theme_intro: Powerful and robust theming system that capable of making any website. Writing your theme using many template engines.
+  theme_link: Check out the Hexo themes
```
Comment (curbengh) [inline themes/navy/languages/en.yml]: intro: Choose from a huge library of spectacular, feature-packed and customizable themes. Create your own using any compatible template engines.
link: Explore themes

Comment (SukkaW) [reply themes/navy/languages/en.yml]: Sounds great!

**Comment by curbengh** [comment]
> But I am not sure what background color should I use in that section.

Alternate between `color-navy` (existing styl variable) and `lighten(color-navy, 10%)` (used in #banner-start-command)?

If one section is `color-navy`, then next section is `lighten(color-navy, 10%)` and so forth.

File: themes/navy/source/css/_partial/index.styl [hunk 2]
```diff
@@ -108,17 +118,79 @@
...
+
+#theme-link
+  color: color-border
+  border-color: color-border
+  &:hover
+    background: @border-color
+    color: #283542
+
+#theme-wrap
+  background: #283542
```
[the remark this answers is not in the document]
Comment (SukkaW) [reply themes/navy/source/css/_partial/index.styl]: Updated

File: themes/navy/languages/zh-cn.yml [hunk 3]
```diff
@@ -9,6 +9,8 @@ menu:
 
 index:
   get_started: 开始使用
+  theme_intro: 在众多精彩、强大、可定制的主题中选择；使用任何兼容的模板引擎创建自己的主题。
```
Comment (curbengh) [inline themes/navy/languages/zh-cn.yml]: 感觉＂美观＂（替代＂精彩＂）比较合适。以＂精彩＂来形容网页设计感觉怪怪的

Comment (SukkaW) [reply themes/navy/languages/zh-cn.yml]: Typo fixed.

**Pull request merged**